web-components/src/components/inputs/new/ImageDrop/ImageDrop.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { forwardRef, useState } from 'react';
```

Comment:
is there a way with react-hook-form to set the field as dirty on change from the input component itself? just like it was the case for the formik based component using `setFieldTouched`
this way we don't have to handle this logic at the form level

Reply:
I'm not sure about that, it might not be possible because we're not supposed to change this manually. The problem we have with this form is that we need to add an empty input field at the initialization which makes the form dirty by default. That's why we can't rely on the `isDirty` for this form specifically but for other forms that don't have this constraint it should be fine to rely on `isDirty` only and we shouldn't need to modify it.